internal/credentials/errors.go

```diff
@@ -44,6 +60,10 @@ type EnvURLCollisionError struct {
 	URL  string
 }
 
+func NewEnvURLCollisionError(name, url string) *EnvURLCollisionError {
```

Comment:
EnvVar credentials related errors to be wiped out on sub-sequent PR, as we talked about removing breadcrumbs of what once was support for environment credentials